stylesheets/styles.css

```diff
@@ -0,0 +1,334 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;700&display=swap');
+:root {
+    --yellow: #E8CEAD;
+    --pink: #E0A1A1;
+    --green: #9DA993;
+    --blue: #E7E8E9;
+}
+* {
+    box-sizing: border-box;
+    font-family: Poppins, sans-serif;
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    height: 100%;
+    width: 100vw;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+}
+
+
+a:visited {
+    color: white
+}
+
+a:hover {
+    color: var(--pink);
+    font-weight: 500;
+}
+/*Navigation Bar*/
+.nav-bar {
+    width: 100vw;
+    background-color: var(--green);
+    margin: 0;
+}
+
+ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    margin: 0;
+    height: 3em;
+}
+
+ul li {
+    padding-right: 1em;
+    font-weight: 300;
+    color: white;
+    margin-right: 2em;
+}
+
+li:first-child {
+    margin-right: auto;
+    margin-left: 2em;
+}
+
+/*Hero Section*/
+.hero {
+    background-color: #cccccc;
+    background-image: url("../images/ben-neale-29w9FiMWSr8-unsplash.jpg");
+    padding-top: 150px;
+    padding-bottom: 150px;
+    background-size: cover;
+}
+
+.intro {
+    margin-left: 2rem;
+}
+
+.intro h3 {
+    font-weight: 700;
+}
+
+#hi-text {
+    color: var(--pink);
+}
+#im-a-text {
+    color: var(--yellow);
+}
+
+.intro h1 {
+    font-size: 4rem;
+    font-weight: 500;
+    color: black;
+}
+
+.intro h2 {
+    font-size: 2rem;
+    font-weight: 400;
+    color: black;
+}
+/*Hero Section Ends*/
+
+/*Home Page Other Pages Starts*/
+.other-pages {
+    display: flex;
+    height: 35vh;
+    padding: 2em 2em; 
+    justify-content: space-around;
+}
+
+.other-pages h3 {
+    color: var(--yellow);
+}
+
+.other-pages h3, p, a {
+    margin-bottom: 1rem;
+}
+
+.link-section {
+    padding: 1rem;
+}
+
+/*Home Page Other Pages Ends*/
+
+/*Footer Starts*/
+.footer {
+    display: flex;
+    width: 100vw;
+    background-color: var(--yellow);
+    margin: 0;
+    display: flex;
+}
+
+.footer p {
+    height: 2em;
+    padding-left: 1em;
+    font-weight: 300;
+    display: flex;
+    align-content: center;
+    font-size: 0.75em;
+}
+
+.footer a {
+    height: 2em;
+    padding-left: 1em;
+    font-weight: 300;
+    align-content: center;
+}
+/*Footer Ends*/
+
+/*Portfolio Starts*/
+.portfolio-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 3fr;
+    background-color: #e7e8e9;
+}
+
+#portfolio-header-section {
+    grid-column: 1 / 3;
+    grid-row: 1 / 2;
+    text-align: center;
+    padding-top: 3rem;
+    padding-bottom: 2rem;
+}
+#portfolio-header-section h1 {
+    font-weight: 300;
+}
+
+#portfolio-header-section h2 {
+    font-weight: 300;
+    font-size: 2rem;
+}
+
+#portfolio-header-section p {
+    font-weight: 300;
+}
```

Comment:
Maybe we can combine these to save space
```css
#portfolio-header-section h1,
#portfolio-header-section h2,
#portfolio-header-section p {
    font-weight: 300;
}

#portfolio-header-section h2 {
    font-size: 2rem;
}
```
It doesn't seem like a lot of space saved, but it does keep elements organized so that you can change a rule one place instead of several palces.